backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/document/model/ElasticsearchIndexDescriptor.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.elasticsearch.document.model;
```

Comment:
I think it would make more sense to mirror the engine packages. So this would be:

```suggestion
package org.hibernate.search.backend.elasticsearch.metamodel;
```

Implementations can stay where they are, that doesn't matter. But the API matters.